packages/rs-drive-abci/src/execution/platform_events/core_based_updates/update_quorum_info/v0/mod.rs

```diff
@@ -173,139 +203,214 @@ where
                 }
             });
 
-        if validator_set_quorum_type == chain_lock_quorum_type {
-            // Remove validator_sets entries that are no longer valid for the core block height
-            if removed_a_validator_set || added_a_validator_set {
+        // Update Chain Lock quorums
+
+        self.update_quorums(
+            QuorumSetType::ChainLock(chain_lock_quorum_type),
+            block_platform_state,
+            platform_state,
+            &extended_quorum_list,
+            is_validator_set_updated,
+            core_block_height,
+        )?;
+
+        // Update Instant Lock quorums
+
+        self.update_quorums(
+            QuorumSetType::InstantLock(instant_lock_quorum_type),
+            block_platform_state,
+            platform_state,
+            &extended_quorum_list,
+            is_validator_set_updated,
+            core_block_height,
+        )?;
```

Comment:
I would do this only if instant_lock_quorum_type is different to chain_lock_quorum_type right?

Reply:
Fixed